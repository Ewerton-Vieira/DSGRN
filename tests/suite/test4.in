dsgrn network ../networks/4D_Model2.txt morsegraph graphviz '[["X",[3,3,"FF8EC8"],[2,0,1]],["Y1",[1,1,"2"],[0]],["Y2",[1,1,"2"],[0]],["Z",[1,1,"2"],[0]]]'
