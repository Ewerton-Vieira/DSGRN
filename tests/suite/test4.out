digraph {
0[label="XC {X, Z}"];
1[label="XC {X, Z}"];
2[label="XC {X, Z}"];
3[label="XC {X, Y1, Y2}"];
4[label="FP { 3, 1, 1, 1 }"];
5[label="FP { 2, 0, 0, 1 }"];
6[label="XC {X, Z}"];
0 -> 1;
1 -> 2;
2 -> 3;
2 -> 6;
3 -> 4;
3 -> 5;
}

